scripts-dev/federation_client.py

```diff
@@ -18,14 +18,21 @@
 from __future__ import print_function
 
 import argparse
+from urlparse import urlparse, urlunparse
+
 import nacl.signing
 import json
 import base64
 import requests
 import sys
+
+from requests.adapters import HTTPAdapter
 import srvlookup
 import yaml
 
+#from httplib import HTTPConnection
```

Comment:
@richvdh stray comment

Reply:
well ish. It was left in to remind me how to enable debugging on the requests lib. Arguably it should be a better comment ...